integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -1946,10 +1946,11 @@ private static void deployApplication(String namespace, String domainUid, String
     assertNotEquals(-1, serviceNodePort, "admin server default node port is not valid");
     logger.info("Deploying application {0} to domain {1} cluster target cluster-1 in namespace {2}",
         clusterViewAppPath, domainUid, namespace);
+    String targets = "{ identity: [ clusters, 'cluster-1' ] }";
     ExecResult result = DeployUtil.deployUsingRest(K8S_NODEPORT_HOST,
```

Comment:
We usually use static import. Static import DeployUtil.deployToClusterUsingRest and change this DeployUtil.deployToClusterUsingRest to deployToClusterUsingRest.

Reply:
done